llama-rs/src/lib.rs

```diff
@@ -95,6 +109,38 @@ pub struct Vocabulary {
     mapping: Vec<Token>,
 }
 
+#[derive(serde::Serialize)]
+/// A serializable snapshot of the inference process. Can be saved to disk
```

Comment:
Full stop at the end, otherwise rustdoc will render this as "Can be saved to disk Useful for prompt caching"